pio/lib/WiFiManagerKT/WiFiManagerKT.cpp

```diff
@@ -691,7 +692,8 @@ void WiFiManager::handleWifi()
   page += FPSTR(HTTP_FORM_START1);
   page += _ssid;
   page += FPSTR(HTTP_FORM_START2);
-  page += _pass;
+  // don't send existing password to web
+  //page += _pass;
```

Comment:
there was a long discussion on this and the desire was to be able to see the password - so lets populate that field. However I am ok to mask that field with asterisks

Reply:
@universam1 Got it. Before I submit that, would it make sense to merge the upstream WifiManager code instead? As @MarcoCLA mentioned that has the fix for displaying the password as asterisks. I'm not sure about specific changes but it does look to be more actively maintained than the current library.

Reply:
There are a couple of issues that I have solved manually, regarding the responsive UI and interactive elements. Would need to see if that is possible nowadays with upstream first, but that would be a longer goal

Reply:
Ok. I've re-enabled sending of the PW to the webUI but it's hidden with asterisks. I'll open another PR for the merge of upstream WifiManager after I have some time to test that.